ayon_api/server_api.py

```diff
@@ -2154,7 +2196,7 @@ def download_addon_private_file(
         # Filename can contain "subfolders"
         dst_dirpath = os.path.dirname(dst_filepath)
         if not os.path.exists(dst_dirpath):
-            os.makedirs(dst_dirpath)
+            os.makedirs(dst_dirpath, exist_ok=True)
```

Comment:
When using `exist_ok` there's no need to check for existence?
```suggestion
         os.makedirs(dst_directory, exist_ok=True)
```